engine/apps/webhooks/models/webhook.py

```diff
@@ -337,3 +339,16 @@ class WebhookResponse(models.Model):
     def json(self):
         if self.content:
             return json.loads(self.content)
+
+
+@receiver(post_save, sender=WebhookResponse)
+def webhook_response_post_save(sender, instance, created, *args, **kwargs):
+    if not created:
+        return
+
+    # TODO: find a better way to filter, additional_settings__isnull is a workaround
```

Comment:
At this point we should have the alert group for which the webhook was triggered, right? and then, the integration too. Can we filter the source channel using the connection model? Could there be more than one source channel connected?

Reply:
changed it to
```python
source_alert_receive_channel = webhook.filtered_integrations.filter(
    connected_alert_receive_channels__isnull=False
).first()
```

which is definitely better than checking by `additional_settings`. I think it'll be possible to have multiple source channels and we'll need to address it, I left a TODO for a future PR.